lib/results_validators/utils.rb

```diff
@@ -12,7 +12,7 @@ module Utils
       ScramblesValidator,
     ].freeze
 
-    VALIDATORS_WITH_FIX = ALL_VALIDATORS.select(&:has_automated_fix?).freeze
+    VALIDATORS_WITH_FIX = ALL_VALIDATORS.select(&:automated_fix?).freeze
```

Comment:
What about `automated_fix_possible?`? Don't feel very strongly about this one though, because it's quite long. Feel free to dismiss.

Reply:
automatically_fixable?